optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js

```diff
@@ -1261,7 +1261,7 @@ jQuery.extend({
 			// the count of uncompleted subordinates
 			remaining = length !== 1 || ( subordinate && jQuery.isFunction( subordinate.promise ) ) ? length : 0,
 
-			// the master Deferred. If resolveValues consist of only a single Deferred, just use that.
+			// the main Deferred. If resolveValues consist of only a single Deferred, just use that.
```

Comment:
I do not think this is a correct replacement.

Reply:
The question is whether we have to change also such an included 3rd party sources.

Reply:
We are replacing the `master` branch.
If we're replacing every occurrence of the word `master`, so be it - but that's for another PR, and a naming discussion. Please remove it from this PR, and I'll approve.

Reply:
Good point, @rsynek. This is not even our source code, therefore subsequent updates would bring the word back in.